CodeGeneration/Sources/SyntaxSupport/AttributeNodes.swift

```diff
@@ -357,7 +365,8 @@ public let ATTRIBUTE_NODES: [Node] = [
     kind: .differentiabilityArgument,
     base: .syntax,
     nameForDiagnostics: "differentiability argument",
-    documentation: "A differentiability argument: either the \"self\" identifier, a function parameter name, or a function parameter index.",
+    documentation:
+      "A differentiability argument: either the \"self\" identifier, a function parameter name, or a function parameter index.",
```

Comment:
I'm fine with this change in general (I think 120 is a nice compromise between 80 and <massive lines>), though it really seems like we need to teach swift-format to split both strings and comments - the split here doesn't really make sense.

Reply:
I couldn't agree more. It would be amazing if `swift-format` could split it in a better way 🥺